tests/multisigNFTs_test.go

```diff
@@ -0,0 +1,192 @@
+		{"multiSig", false, false, "[case-2.0] Create MultiSig Tx for NFTs [Transfer-token-ownership] - [internalTx3x_transferTokenOwnership_err3] - wait-15-seconds.", "multisig-acc-nft-2", "100000000cin", 0, MultisigInfo{"create-internal-tx", "", "", 0, nil, "nft-grp-addr-2", 9, internalTx3x_transferTokenOwnership_err3}, "MEMO : xxxxx", nil}, // nx "sequence":
+		{"multiSig", false, false, "[case-2.0] Sign MultiSig Tx for NFTs [Transfer-token-ownership] - submit which signed by multisig-acc-nft-3 - wait-15-seconds.", "multisig-acc-nft-3", "100000000cin", 0, MultisigInfo{"multiSig-sign-tx", "", "", 0, nil, "nft-grp-addr-2", 7, internalTx3x_transferTokenOwnership1}, "MEMO : Sign MultiSig Tx", nil},
+		{"nonFungibleToken", false, false, "[case-2.0] VERIFY nonfungible token transfer ownership [TNFT-PUBLIC-FALSE-02] - Happy path - wait-15-seconds.", "nft-mostafa", "0cin", 0, NonFungibleTokenInfo{"verify-transfer-token-ownership", "", "", "", "TNFT-PUBLIC-FALSE-02", "nft-mostafa", "nft-grp-addr-2", "token metadata", "", "", "", true, false, true, true, false, "nft-jeansoon", "0", "nft-carlo", "", "APPROVE_TRANFER_TOKEN_OWNERSHIP", "", "", []string{""}}, "[case-2.0] VERIFY nonfungible token transfer ownership [TNFT-PUBLIC-FALSE-02] - Happy path", nil},
+
+		// Need signed by : {"multisig-acc-nft-2", "multisig-acc-nft-3", "multisig-acc-nft-4"}, owner=="multisig-acc-nft-1"
+		// Threshold == 3, under 'nft-grp-addr-4'
+		{"multiSig", false, false, "[case-2.0] Create MultiSig Tx for NFTs [Accept-token-ownership] - submit counter+0 - Happy Path - wait-15-seconds.", "multisig-acc-nft-2", "100000000cin", 0, MultisigInfo{"create-internal-tx", "", "", 0, nil, "nft-grp-addr-4", 0, internalTx3x_acceptTokenOwnership1}, "MEMO : xxxxx", nil},                // nx "sequence":
+		{"multiSig", false, false, "[case-2.0] Create MultiSig Tx for NFTs [Accept-token-ownership] - [internalTx3x_acceptTokenOwnership_err1] - wait-15-seconds.", "multisig-acc-nft-4", "100000000cin", 0, MultisigInfo{"create-internal-tx", "", "", 0, nil, "nft-grp-addr-4", 1, internalTx3x_acceptTokenOwnership_err1}, "MEMO : xxxxx", nil}, // nx "sequence":
+		{"multiSig", false, false, "[case-2.0] Create MultiSig Tx for NFTs [Accept-token-ownership] - [internalTx3x_acceptTokenOwnership_err3] - wait-15-seconds.", "multisig-acc-nft-4", "100000000cin", 0, MultisigInfo{"create-internal-tx", "", "", 0, nil, "nft-grp-addr-4", 2, internalTx3x_acceptTokenOwnership_err3}, "MEMO : xxxxx", nil}, // nx "sequence":
+		{"multiSig", false, false, "[case-2.0] Sign MultiSig Tx for NFTs [Accept-token-ownership] - submit which signed by multisig-acc-nft-4 - wait-15-seconds.", "multisig-acc-nft-4", "100000000cin", 0, MultisigInfo{"multiSig-sign-tx", "", "", 0, nil, "nft-grp-addr-4", 0, internalTx3x_acceptTokenOwnership1}, "MEMO : Sign MultiSig Tx", nil},
```

Comment:
Need to wait for 15 seconds?

Reply:
If I am not mistaken only for approve process we need wait. Fo other transactions we don't have to wait.

Reply:
Hi, Mostafa

Will test on this and revert.

Reply:
Please see below latest update.